src/uhttp.c

```diff
@@ -231,12 +234,17 @@ static int process_header_line(const unsigned char* buffer, size_t len, size_t*
                 // Remove leading spaces
                 while (*targetPos == 32) { targetPos++; }
 
-                size_t valueLen = (&buffer[index])-targetPos;
-                char* headerValue = (char*)malloc(valueLen+1);
-                if (headerValue == NULL)
+                size_t valueLen = safe_subtract_size_t((&buffer[index]), targetPos);
+                size_t malloc_size = safe_add_size_t(valueLen, 1);
+
+                char* headerValue;
+                if (malloc_size == SIZE_MAX ||
+                    (headerValue = (char*)malloc(malloc_size)) == NULL)
                 {
+                    LogError("Cannot malloc memory, size:%zu", malloc_size);
```

Comment:
Could the message be a little more specific please, for systems where the files and line numbers are not printed?

Reply:
```suggestion
                    LogError("Cannot malloc headerValue, size:%zu", malloc_size);
```